bigquery/google/cloud/bigquery/client.py

```diff
@@ -1239,6 +1239,31 @@ def insert_rows_json(self, table, json_rows, row_ids=None,
 
         return errors
 
+    def list_partitions(self, table, retry=DEFAULT_RETRY):
+        """List the partitions in a table.
+
+        :type table: One of:
```

Comment:
Please update to [Google style docstring](http://www.sphinx-doc.org/en/stable/ext/example_google.html)